dash-pipeline/dockerfiles/DOCKER_VPP_IMG.env

```diff
@@ -0,0 +1,5 @@
+# Define docker image repo/name:tag
+# Changing this will cause build/publish to occur in CI actions
+export DASH_ACR_REGISTRY=sonicdash.azurecr.io
+export DOCKER_VPP_IMG_NAME?=${DASH_ACR_REGISTRY}/dash-vpp-bldr
+export DOCKER_VPP_IMG_CTAG?=240118
```

Comment:
We generally use a date code as the tag, this should be updated to 240814.

Reply:
updated